packages/expo-media-library/src/MediaLibrary.ts

```diff
@@ -145,6 +145,63 @@ export type AssetInfo = Asset & {
    * @platform ios
    */
   orientation?: number;
+  /**
+   * This field is available if the type is an image, and the subtype is livePhoto
+   * Whether the asset is stored on the network (iCloud on iOS).
+   * @platform ios
+   */
+  pairedVideoAsset?: PairedVideoAsset | null;
+};
+
+/**
+ * Represents a video component of a Live Photo on iOS. This is the paired video asset that
+ * accompanies a Live Photo's still image, containing the motion content.
+ */
+export type PairedVideoAsset = {
```

Comment:
I guess we can remove some properties by merging it with the base type
```suggestion
export type PairedVideoAsset = Asset & {
```

Reply:
Yeah, that's a very good point. I was so focused on what it looked like in the ImagePicker that I missed this. This sounds reasonable.